python/agents/application/trade.py

```
@@ -0,0 +1,76 @@
+        self.polymarket = Polymarket()
+        self.gamma = Gamma()
+        self.agent = Agent()
+
+    def pre_trade_logic(self) -> None:
+        self.clear_local_dbs()
+
+    def clear_local_dbs(self) -> None:
+        try:
+            shutil.rmtree("local_db_events")
+        except:
+            pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)